src/Vlingo.Xoom/actors/Settings.cs

```diff
@@ -11,34 +11,33 @@
 using System.Linq;
 using System.Reflection;
 
-namespace io.vlingo.xoom.actors {
+namespace Vlingo.Xoom.Actors {
     public class Settings {
 
-        private static Dictionary<string, string> PROPERTIES = new Dictionary<string, string>();
-        private static readonly string PROPERTIES_FILENAME = "/vlingo-xoom.properties";
-        private static readonly long serialVersionUID = 1L;
-        private static readonly Dictionary<object, object> DEFAULT_DATABASE_PROPERTIES = new Dictionary<object, object>() {
+        private static Dictionary<string, string> properties = new Dictionary<string, string>();
+        private static string propertiesFileName = "/vlingo-xoom.properties";
+        private static Dictionary<object, object> defaultDatabaseProperties = new Dictionary<object, object>() {
             { "database", "IN_MEMORY" },
             { "query.database", "IN_MEMORY" }
         };
 
         static Settings() {
-            loadProperties();
+            LoadProperties();
         }
 
-        public static void loadProperties() {
+        public static void LoadProperties() {
```

Comment:
Properties format should be as follows  https://github.com/vlingo-net/vlingo-net-http/blob/master/src/Vlingo.Http.Tests/vlingo-actors.json

We don't have yet loading `.json` file from `GetManifestResourceStream` but we could add that here https://github.com/vlingo-net/vlingo-net-common/blob/master/src/Vlingo.Common/ConfigurationProperties.cs. Leave it for the moment here but make it work with the format used by `.json` files and `ConfigurationProperties` class from `vlingo-net-common`

Reply:
Fixed